scenarios/aws/dockerVM/run.go

```diff
@@ -48,6 +49,12 @@ func Run(ctx *pulumi.Context) error {
 			}
 		}
 		agentOptions = append(agentOptions, dockeragentparams.WithAgentServiceEnvVariable("DD_LOG_LEVEL", pulumi.String("debug")))
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can probably remove the log debug level here.